lib/engine/config/game/g_18_co-full_b_group.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
This file could be removed. It contains the full details of the actual 18CO private companies. Only one company from "group B" is added to the game. I was able to accomplish this with a single existing private / ability combo. We lose out on the flavor though.

Reply:
please delete this file